### DISCUSSION TITLE
Spack + Dakota 6.18

### DISCUSSION THREAD

**Discussion by tdrayna** [snl-dakota/dakota#62] [Show and tell]
I was able to successfully update spack to build Dakota 6.18 using the public GitHub repository. If anyone is interested or can review my PR please see this:

[PR#39882](https://github.com/spack/spack/pull/39882)

Thanks!
Travis

**Comment by jadamstephens** [comment]
Travis,

Thank you! We are very novice spack users, so I doubt my review would be useful. I will say that nothing looks obviously suspicious to me in the changeset. We definitely appreciate you making this available.

Adam
